### DISCUSSION TITLE
SEB installation error version 3.3.0.354 0x80072ee7

### DISCUSSION THREAD

**Discussion by neluwala** [SafeExamBrowser/seb-win-refactoring#200] [Q&A]
We get this error when installing SEB 3.3.0.354
There is sufficient space in C drive
![seb erro2](https://user-images.githubusercontent.com/10020525/133706230-2b479049-e359-44e4-99a9-c8d30e3bbdb6.jpeg)

Please advise how to solve this issue
Thank you

**Comment by dbuechel** [comment]
Try to install the Microsoft Edge WebView2 Runtime manually: https://go.microsoft.com/fwlink/p/?LinkId=2124703

**Reply by souviksenapati** [reply]
I am also getting the same error after manually installing all Microsoft Edge WebView2 Runtime, VC++ 2015-2019, and my system has. Netframework 6.0 installed along with all inbuilt versions of .Net please help I have an exam next week and the exam website support only SEB v3.3.0.354. I can easily install SEB v3.3.1 but this version is not supported by the website.
[Safe_Exam_Browser_20211215000337.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7717716/Safe_Exam_Browser_20211215000337.log)
![Error](https://user-images.githubusercontent.com/47359760/146154155-0f9a749b-ff77-48f4-92d5-a55f383a2588.png)

**Reply by dbuechel** [reply]
You'll need to manually install all dependencies and then use the MSI setup.

**Reply by TaimurShahikhel** [reply]
I am having exactly the same issue I have been trying to install SEB 3.3.0 but despite having manually installed all the applications its still not working. Please help me I really need it. Anyone knows the solution?

**Comment by souviksenapati** [comment]
I did but still setup failed

**Comment by npierson** [comment]
Hello, the exact same issue here.  We give exams to a few hundred users a week, and starting Monday this Microsoft error came up for multiple users on different WIndows machines in different schools.  We suggested they manually update Windows with the Edge WebView2 Runtime, the .NET update, and the C++ links they suggest but it did not work.   It is especially odd because our users have been using SEB 3.3.0.354 since August and suddenly we are getting these errors as of 12/13/2021.

Is this resolved with the version that was released in October?

![aquifer](https://user-images.githubusercontent.com/5138923/146238485-050e8041-d81a-40f3-8c75-c3650278ee29.jpg)

**Reply by dbuechel** [reply]
Yes, we always recommend using the latest official release version. When installing the dependencies manually, you'll then have to use the MSI setup as it can occur that the bundle in either case won't work due to a failing dependency installation.

**Comment by TaimurShahikhel** [comment]
I am having exactly the same issue I have been trying to install SEB 3.3.0 but despite having manually installed all the applications its still not working. Please help me I really need it. Anyone knows the solution?

**Reply by dbuechel** [reply]
Are you using the correct MSI setup for your platform (either ..._x64_Setup.msi or ..._x86_Setup.msi) after installing all the required dependencies manually?

All required dependencies at the time of writing are:

- .NET Framework 4.7.2 Runtime: https://dotnet.microsoft.com/download/dotnet-framework/net472
- Microsoft Edge WebView2 Runtime: https://go.microsoft.com/fwlink/p/?LinkId=2124703
- Visual C++ 2015-2019 Redistributable: https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

**Reply by JubayerSarkar** [reply]
It don't work still, and My Computer has .Net 4.8 advance, so it won't install 4.7.2, otherwise i installed all the other dependencies, but it shows the same errror Element not found, As i need to take Tolc exam which doesn't support any other version of SEB than 3.3.0.354, i really need to install this version. Please do something. We need a solution ASAP.

**Reply by dbuechel** [reply]
As explained above, once you have installed the dependencies, you need to install the **MSI setup** (_not_ the setup bundle), i.e. either one of these two depending on the architecture of your system:

https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/3.3.0/SEB_3.3.0.354_x64_Setup.msi
https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/3.3.0/SEB_3.3.0.354_x86_Setup.msi

**Comment by fuleo** [comment]
Hey guys, I just want to echo this. I've been struggling with the installation of this on a few computer in the labs. I just realized my issue is related to dotnet35 installations, even though Windows listed dotnet35 as already installed. 

Here are the steps to replicate the solution.

1) Download GTools from http://p-nand-q.com/download/gtools/pathed.html
2) Install GTools
3) Run -> pathed.exe which will induce the reinstallation of dotnet35.
4) Restart
5) Try to install safe exam browser. It should work now! 


<img width="639" alt="Screenshot 2022-02-08 at 3 34 44 PM" src="https://user-images.githubusercontent.com/809763/152939901-42d58ae3-ef2b-43bf-9bd3-8858fa9e9081.png">


Hope this helps you.

**Reply by dbuechel** [reply]
Thanks for your input. Would you mind sharing to which SEB version your solution relates? For SEB 3.x, installing the .NET Framework 4.7.2 Runtime should be sufficient to the best of my knowledge.

**Reply by fuleo** [reply]
Hello dbuechel, we tried version safe exam browser 3.2 and 3.3 (latest).

Indeed, we've checked our system and we had dotnet 4.7.2 installed on our PCs, yet we keep getting the error during installation of safe_exam_browser. 

In fact if we download the dot3.5 installer, we couldn't install it. Windows 10 would say it's already installed or there's a newer version 4.7.2 and I don't need to install it.

Only by running that above mentioned command were we induced with the prompt to install the *supposedly* missing dotnet 3.5 library.

**Reply by dbuechel** [reply]
Okay, thanks for the clarification. That's really weird indeed, but let's hope this helps other users who might run into the same issue.
